README.md

```diff
@@ -48,6 +48,15 @@ Without Bundler:
 gem install postmark
 ```
 
+## Deprecations
+
+Deprecations can be either silenced or set to raise using:
+
+```rb
+Postmark::Deprecations.behavior = :silence
+Postmark::Deprecations.behavior = :raise
```

Comment:
This might be partially a pet peeve, but one thing I consider a smell in a project, is seeing warnings output whenever running the app.

In my experience, it conditions you into thinking "well, it's OK to have some errors", and you start not seeing _actual_ problems.

Of course, best thing to do is to fix them! 💡 ...but each team might have their own priorities, reasons it can't happen right away, and different ways of handling that while they "get around to it". Some teams might be more aggressive about fixing things like this, and would like deprecations to fail the build.

Mature deprecation frameworks provide those options, and I think users of this gem should have that freedom and not be forced to see warnings that we emit. 

That includes us and our test suite! I would hate to see those warnings whenever we run the suite.

Hope that makes sense!